.github/workflows/deploy-personal-dev-environment.yml

```diff
@@ -179,7 +187,11 @@ jobs:
         run: |
           cd data-dashboard-infra
           source uhd.sh
-          uhd docker build ingestion ${{ inputs.name }}
+          if [[ "${{ inputs.backend_branch }}" == "main" ]]; then
```

Comment:
I"ll update this in a follow up PR so that we can deploy the ingestion image seperately from the backend image